secrets/testing.go

```diff
@@ -0,0 +1,94 @@
+package secrets
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+
+	"github.com/reddit/baseplate.go/filewatcher"
+)
+
+const (
+	// JWTPubKeyPath is the expected key for the EdgeRequestContext public
+	// key.
+	JWTPubKeyPath = "secret/authentication/public-key"
+
+	testVaultURL   = "vault.reddit.ue1.snooguts.net"
+	testVaultToken = "17213328-36d4-11e7-8459-525400f56d04"
+)
```

Comment:
nit: I would prefer to separate exported and unexported consts into different const blocks.